README.md

```diff
@@ -7,71 +7,120 @@
 
 **Looking for your help!**
 
-For Rails Girls Summer of Code we are planning to build a simple app that
-aggregates daily status updates, commit activity, GitHub issues and other
-things into an activity stream.
+For Rails Girls Summer of Code building an app that
+aggregates daily status updates, commit activity, GitHub issues, and other
+things into an activity stream. The app also supports coaches and organizers
+in centralizing the information they need to have to help keep RGSoC 
+running along smoothly. 
 
 Main goals are:
 
-* make it easy to get an overview of activity/progress for each of the teams for supervision
-* make it easy for interested remote coaches to find opportunities to give support/help
+* make it easy to get an overview of activity/progress for each of the teams 
+for supervision:
+* make it easy for interested remote coaches to find opportunities to give 
+support/help
+* keep administrative information and processes in one place
 * display to the world how much amazing stuff is happening
 
 We are planning to require teams to keep a daily log of short updates about
 their work. Our idea is to allow any sort of blog type tool for that (maybe
 recommend a few) and aggregate things through RSS in a central app. This app
 could then act as a webhook target for GitHub events.
 
-Since we still are somewhat overwhelmed with the amount of work we'd like to
-ask the community for help with this. The app would need to be available
-(initial, basic version) on 1st of July, ideally a few days earlier.
+We're really excited at how much this app has grown and developed, and are really 
+grateful for all of the community help along the way. There's still plenty to do 
+here - and we'd love to have your contributions! Hop on over to the 
+[issues](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues) 
+and have a look around, or open up a 
+[new issue](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/new).
 
 Features:
 
 * Users can sign in through GitHub Oauth
-* They can create and update teams
-* Teams have members (students, coaches, mentors aka project maintainers), GitHub repositories, a log URL
+* Users can create and update teams (assuming they have the 
+right access rights)
+* Teams have members (students, coaches, mentors aka project maintainers),
+ GitHub repositories, a log URL
```

Comment:
I honestly can't remember whether the GH stuff goes into the daily status update (probably not though? I'd have to check) but maybe if that's the case filing everything under “daily log with status updates” makes the most sense. 👍